### PR TITLE
Add incentive banner and copy tweaks

### LINKED ISSUE #6836: T - Implement varied callout to swag and special event invitations for non-subscribers/subscribers.
See #6835 description.

*Description*
Depending on whether subscriber or not - user should see incentive callout.

### PR DESCRIPTION
**Pull request by schalkneethling**
This adds the incentives to the payments landing and thank pages. It also adds the relevant copy tweaks to the edit profile page:

## Edit profile, active subscriber

Screenshot 2020-04-14 at 14 16 09

## Edit profile, non-subscriber

Screenshot 2020-04-14 at 14 17 51

## Mobile

Screen Shot 2020-04-14 at 14 19 26-fullpage

## Tablet

Screenshot 2020-04-14 at 14 20 53

## Desktop, active subscriber

Screenshot 2020-04-14 at 14 16 30

## Desktop, non-subscriber

Screenshot 2020-04-14 at 14 16 39

@mindy @malqinneh r?

fix #6836

### CODE REVIEW

File: kuma/javascript/src/payments/incentives.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                            />
+                        )}
+                    </p>
+                </li>
+                <li className="invitations">
+                    <h4>{gettext('Invitations to events')}</h4>
+                    <p>
+                        {!isSubscriber && nonSubscriberInvitationCopy}
+
+                        {isSubscriber && subscriberInvitationCopy}
```
Comment (schalkneethling) [inline kuma/javascript/src/payments/incentives.jsx:60]: I guess using a ternary here would make more sense?

**Comment by malqinneh** [comment]
Thanks @schalkneethling! Looking good!

File: kuma/javascript/src/payments/incentives.jsx (hunk 1 above)
Comment (schalkneethling) [reply kuma/javascript/src/payments/incentives.jsx]: I have gone the ternary route. Let me know if that was the wrong call.

**Comment by schalkneethling** [comment]
> Thanks @schalkneethling! Looking good! Can you add a period at the end of the first incentive description?
> 
> ie.
> image
> 
> Also, can you update the swag incentive copy and code to be 20% / MDNMEMBER20?

Done. Also, what is the URL to the merch store? :)

**Comment by malqinneh** [comment]
> Done. Also, what is the URL to the merch store? :)

Not officially launched but the current URL is: https://shop.spreadshirt.com/mozilla-developer-network/

**Pull request merged**